scripts/silo/add_silo_decorators.py

```diff
@@ -62,94 +61,62 @@ def read_audit():
     return list(parse_audit(silo_audit))
```

Comment:
There can be leading warnings in getsentry with braces. I've been using `pipe_input.index('{"')` but maybe there's a better way to detect the start of the data.
```
INFO:The Sentry runner will report development issues to Sentry.io. Use SENTRY_DEVENV_NO_REPORT to avoid reporting issues.
Using configuration 'getsentry.conf.settings.dev'
21:28:41 [WARNING] sentry.utils.geo: settings.GEOIP_PATH_MMDB not configured. (logging.googleapis.com/labels={'name': 'sentry.utils.geo'} severity='WARNING')
21:28:42 [INFO] sentry: flagr.not-configured (logging.googleapis.com/labels={'name': 'sentry'} severity='INFO')
21:28:43 [INFO] sentry.plugins.github: apps-not-configured (logging.googleapis.com/labels={'name': 'sentry.plugins.github'} severity='INFO')
```

Reply:
Yeah, it feels like there should be a better way but 🤷